app/models/custom_notification.rb

```diff
@@ -0,0 +1,31 @@
+class CustomNotification < ApplicationRecord
+
+  belongs_to :organization
+
+  has_and_belongs_to_many :users
```

Comment:
I'm not sure I understand this. Isn't the one that represents massivity `MassiveJob`? shouldn't this relationship be there?

Reply:
DONE!